nf_core/__main__.py

```diff
@@ -34,6 +35,11 @@ def run_nf_core():
     stderr.print("[green]                                          `._,._,'\n")
     stderr.print("[black]    nf-core/tools version {}\n\n".format(nf_core.__version__))
 
+    if not os.environ.get('NFCORE_NO_VERSION_CHECK', False):
+        is_outdated, current_version, remote_version = nf_core.utils.check_if_outdated()
+        if is_outdated:
+            stderr.print("Note: nf-core is out of date; latest version: {}, current version: {}\n\n".format(remote_version, current_version))
```

Comment:
We should wrap this whole block with a `try` / `catch` for any exception types (base `catch Exception`), as we don't really care if it fails and don't want to stop execution..